gemrb/GUIScripts/bg2ee/Start.py

```diff
@@ -0,0 +1,33 @@
+# GemRB - Infinity Engine Emulator
+# Copyright (C) 2023 The GemRB Project
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+def OnLoad():
+	global skip_videos
+
```

Comment:
I know it's a copy, but the global isn't needed. We can just replace the one use with the direct call to GetVar.